static/js/enketo/widgets/notewidget.js

```diff
@@ -63,6 +69,24 @@ define( function( require, exports, module ) {
         /* jshint unused:false */
     };
 
+    function applyLiveLinkHtml( $el ) {
+        var html = $el.html();
+
+        html = html.replace( /\[([^\]]*)\]\(([^)]*<[^>]*\>[^)]*)\)/gm,
+                '<a class="live-link" href="#" target="_blank">$1<span class="href">$2</span></a>' );
```

Comment:
There's a `.hidden` in bootstrap, and another one in `build/enketo.less`. Whichever one wins is fine :) 
If that doesn't work then yeah leave it.